src/main/java/com/rusticisoftware/cloud/v2/client/api/ApplicationManagementApi.java

```diff
@@ -349,7 +349,7 @@ public SettingListSchema getApplicationConfiguration(String learningStandard, Bo
     return apiClient.invokeAPI(localVarPath, "GET", localVarQueryParams, localVarPostBody, localVarHeaderParams, localVarFormParams, localVarAccept, localVarContentType, localVarAuthNames, localVarReturnType);
       }
   /**
-   * 
+   * Get a list of all applications for the realm.
```

Comment:
Ok, maybe it is a result of code generation because it seems to be happening in multiple files.